spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -161,4 +230,79 @@
       end
     end
   end
+
+  describe "#create_as_recurrent", vcr: true do
+   let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "monthly" } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.